README.md

```diff
@@ -305,6 +302,18 @@ node index.js transfer-eth-erc20-from-near --amount 1 --near-sender-account rain
 
 You should observe the change of the ERC20 balance as reported by the CLI.
 
+## Contract Development Workflow
+
+Above steps are ways to run a local bridge and development workflows you need if make any changes to rainbow-bridge-cli. If you want to update any of solidity or rust contracts, they're not in this repo now and workflow is as following.
+
+- Install dependencies, doing `index.js clean`, `prepare`, `start near-node` `start ganache`
```

Comment:
I suggest the following format, notice that bash commands have separate large boxes that makes them easy to copy:


- Install dependencies:
```bash
node index.js clean
node index.js prepare
```
- Start local NEAR network and Ganache
```bash
node start near-node
node start ganache
```